swig/python/README.rst

```diff
@@ -36,35 +36,54 @@ Once you have Anaconda or Miniconda installed, you should be able to install GDA
 
 ``conda install -c conda-forge gdal``
 
-Unix
-~~~~
-
-The GDAL Python bindings requires setuptools.
 
 pip
 ~~~
 
-GDAL can be installed from the `Python Package Index <https://pypi.org/project/GDAL>`__:
+GDAL can be installed from the `Python Package Index <https://pypi.org/project/GDAL>`__.
+Due to the complex nature of GDAL and its components, different bindings may require additional packages installation steps.
+The base GDAL package contains support for OGR, OSR, and GDAL vectors:
 
 ::
 
     pip install gdal
 
-It will be necessary to have libgdal and its development headers installed
-if pip is expected to do a source build because no wheel is available
-for your specified platform and Python version.
 
-To install with numpy support, you need to require the optional numpy component:
+In order to enable raster support, libgdal and its development headers must be installed as well as the Python packages numpy, setuptools, and wheel.
```

Comment:
"raster support". Well GDAL without numpy does support raster, it is just that it is not very convenient if you need to inspect array values, but you can for example script raster conversion with gdal.Translate() without requiring numpy. Maybe use "numpy-based raster support" ? (here and below). The sentence "The base GDAL package contains support for OGR, OSR, and GDAL vectors:" above should also be reworked, as it gives the impression that there's no raster support at all